docker/sawtooth-dev-python

```diff
@@ -86,8 +86,9 @@ RUN echo "deb http://repo.sawtooth.me/ubuntu/ci xenial universe" >> /etc/apt/sou
  && pip3 install \
     pylint \
     bandit \
-    pyformance \
-    coverage --upgrade
+    coverage --upgrade \
```

Comment:
Remove this change, it is already correct after we add the built deb to the repo.